scripts/scil_tractogram_commit.py

```diff
@@ -337,12 +341,17 @@ def main():
     # === Verifications ===
     assert_inputs_exist(parser, [args.in_tractogram, args.in_dwi,
                                  args.in_bval, args.in_bvec],
-                        [args.in_peaks, args.in_tracking_mask])
+                        [args.in_peaks, args.in_tracking_mask,
+                         args.reference])
     assert_output_dirs_exist_and_empty(parser, args, args.out_dir,
                                        optional=args.save_kernels)
     _, ext = os.path.splitext(args.in_tractogram)
-    if ext == '.trk':
-        assert_headers_compatible(parser, [args.in_tractogram, args.in_dwi])
+    if ext in ['.trk', '.tck']:
+        assert_headers_compatible(parser, [args.in_tractogram, args.in_dwi],
+                                  reference=args.reference)
+
+        if ext == '.tck':
+            logging.warning("Commit only works with .trk format")
```

Comment:
If it doesn't work with tck file why do you continue ?

Reply:
Oups sorry. Left over from my first tries.